src/monitor.rs

```diff
@@ -113,6 +113,15 @@ pub trait MonitorHandleProvider: AsAny + fmt::Debug + Send + Sync {
     #[cfg_attr(not(web_platform), doc = "detailed monitor permissions.")]
     fn position(&self) -> Option<PhysicalPosition<i32>>;
 
+    /// Returns physical size of monitor in millimeters.
+    ///
+    /// Returns `None` if the size of the monitor could not be determined.
+    ///
```

Comment:
Could you add a note saying that even if the size could be obtained, it may not be correct and the use should be done with care.

Reply:
Agreed, it seems like this can really only be treated as a hint, and that there's many cases where it might return something confusing (e.g. when the screen is rotated).

Reply:
We use this approach to figure out DPR on x11 in some cases, it sometimes returns bogus metrics, such as `0x0` and so on.